runatlantis.io/docs/server-side-repo-config.md

```diff
@@ -97,16 +100,17 @@ workflows:
 ## Use Cases
 Here are some of the reasons you might want to use a repo config.
 
-### Requiring PR Is Approved Before Apply
+### Requiring PR Is Approved Before Apply or Import
```

Comment:
If we add a future subcommand then this title may turn into

```suggestion
### Requiring PR Is Approved Before Apply or Import or NewSubcommand
```

Can we instead make this simply

```suggestion
### Requiring PR Is Approved Before an applicable subcommand
```

Or something similarly generic?